Code/GraphMol/MolStandardize/MolStandardize.cpp

```diff
@@ -291,7 +366,7 @@ void reionizeInPlace(RWMol &mol, const CleanupParameters &params) {
   std::unique_ptr<Reionizer> reionizer{reionizerFromParams(params)};
   reionizer->reionizeInPlace(mol);
 }
-void reionizeInPlace(std::vector<RWMol *> &mols,int numThreads,
+void reionizeInPlace(std::vector<RWMol *> &mols, int numThreads,
                      const CleanupParameters &params) {
   std::unique_ptr<Reionizer> reionizer{reionizerFromParams(params)};
   auto sfunc = [&](RWMol &m, const CleanupParameters &) {
```

Comment:
Here's there is no need to capture anything.

Reply:
actually, it needs to capture `reionizer`, but only that